node/src/chain_spec.rs

```diff
@@ -155,3 +157,72 @@ fn testnet_genesis(
 		}),
 	}
 }
+
+/// Properties for Litentry.
+pub fn litentry_properties() -> Properties {
+	let mut properties = Properties::new();
+
+	properties.insert("ss58Format".into(), 31.into());
+	properties.insert("tokenDecimals".into(), 12.into());
+	properties.insert("tokenSymbol".into(), "LIT".into());
+
+	properties
+}
+
+pub fn litentry_config() -> Result<ChainSpec, String> {
+	let wasm_binary = WASM_BINARY.ok_or("Development wasm binary not available".to_string())?;
+
+	Ok(ChainSpec::from_genesis(
+		// Name
+		"Litentry",
+		// ID
+		"Litentry",
+		ChainType::Live,
+		move || testnet_genesis(
+			wasm_binary,
+			// Initial PoA authorities
+			vec![
+				// 49xBdg1G5MBrtPwPDK7Wv9WLjbRHZ71iRtAs3pRUPMDLVocf
+				(
+					hex!["9ce8c6f2c22502322fb29a1af5de753ce2c62e9eeb0a93efe1bd0ad56438e93a"].unchecked_into(),
```

Comment:
Any other options rather than using hex format?
Who holds these keys?
And better to read a configuration file?

Reply:
open issue to define the procedure, these accounts just for example.